pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -131,6 +131,13 @@ func (cgc *containerGC) removeOldestN(containers []containerGCInfo, toRemove int
 // removeSandbox removes the sandbox by sandboxID.
 func (cgc *containerGC) removeSandbox(sandboxID string) {
 	glog.V(4).Infof("Removing sandbox %q", sandboxID)
+	// In normal cases, kubelet should've already called StopPodSandbox before
+	// GC kicks in. To guard against the rare cases where this is not true, try
```

Comment:
suggest providing an example of when exactly this might be required, so it doesn't sound like we're just throwing calls in to be doubly and triply sure